Vectra/vectra_cognito_detect/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,126 @@
+[
+  {
+    "value": "{host.ip} set {vectra.detection.name}",
+    "conditions": [      
+      {
+         "field": "event.action",
+         "value": "INFO"
+      },
+      {
+         "field": "host.ip",
+      },
+      {
+         "field": "vectra.detection.name",
+      }      
+    ],
+    "relationships": [
+      {
+        "source": "host.ip",
+        "target": "destination.ip",
+        "type": "set"
+      }
+    ]
+  },
+  {
+    "value": "{vectra.history.host_access.name} logged on to {observer.ip} with {vectra.detection.last_type}",
+    "conditions": [
+      {
+         "field": "vectra.history.host_access.name",
+      },
+      {
+         "field": "observer.ip",
+      },
+      {
+         "field": "vectra.detection.last_type",
+      }  
+    ],
+    "relationships": [
+      {
+        "source": "vectra.history.host_access.name",
+        "target": "observer.ip",
+        "type": "logged on to"
+      }
+    ]
+  },
+    {
+    "value": "{observer.ip} connect to {host.ip} with {vectra.detection.last_type} and Threat {vectra.risk_score_norm}",
+    "conditions": [
+      {
+         "field": "observer.ip",
+      },
+      {
+         "field": "host.ip",
+      },
+      {
+         "field": "vectra.detection.last_type",
+      },
+      {
+         "field": "vectra.risk_score_norm",
+      }        
+    ],
+    "relationships": [
+      {
+        "source": "observer.ip",
+        "target": "host.ip",
+        "type": "logged on to"
+      }
+    ]
+  },
+    {
+    "value": "{vectra.detection.reason} to {vectra.destination.name} from {source.ip} to {destination.ip} to {action.name} {vectra.campaign.link}",
+    "conditions": [
+      {
+         "field": "vectra.campaign.link"
+      },
+      {
+         "field": "action.name"
+      },
+      {
+         "field": "vectra.detection.reason",
+      },
+      {
+         "field": "vectra.destination.name",
+      },
+      {
+         "field": "source.ip",
+      },
+      {
+         "field": "destination.ip",
+      }   
+    ],
+    "relationships": [
+      {
+        "source": "source.ip",
+        "target": "destination.ip",
+        "type": "vectra.detection.reason"
```

Comment:
A field to be used in the property `type`, only text.

Reply:
#457

Now line 43